api/src/main/java/org/pmiops/workbench/api/CloudTaskUserController.java

```diff
@@ -49,73 +55,76 @@ public class CloudTaskUserController implements CloudTaskUserApiDelegate {
           "762320479256"); // terra_prod_aou_prod_2
   private static final Logger log = Logger.getLogger(CloudTaskUserController.class.getName());
 
-  private final UserDao userDao;
```

Comment:
we prefer that Controllers should not directly access DAOs.  I added an equivalent call in `UserService` in this PR, so the last usage of `userDao` was no longer needed.